services/prometheus/scrape_configs_test.go

```diff
@@ -813,59 +813,67 @@ func TestScrapeConfig(t *testing.T) {
 				AWSSecretKey: pointer.ToString("wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"),
 				ListenPort:   pointer.ToUint16(12345),
 			}
+			agent2 := &models.Agent{
+				AgentID:      "/agent_id/agent2",
+				AgentType:    models.RDSExporterType,
+				PMMAgentID:   pointer.ToString("pmm-server"),
+				NodeID:       pointer.ToString("/node_id/node1"),
+				AWSAccessKey: pointer.ToString("AKIAIOSFODNN7EXAMPLE"),
+				AWSSecretKey: pointer.ToString("wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"),
+				ListenPort:   pointer.ToUint16(12345),
+			}
+			agent3 := &models.Agent{
+				AgentID:      "/agent_id/agent3",
+				AgentType:    models.RDSExporterType,
+				PMMAgentID:   pointer.ToString("pmm-server"),
+				NodeID:       pointer.ToString("/node_id/node1"),
```

Comment:
Why we would have several configurations for the same node? That test is not illustrative at all.